packages/msw/mixin.core.js

```diff
@@ -0,0 +1,110 @@
+const { Mixin } = require('hops-mixin');
+
+const getMockServiceWorkerContent = () => {
+  const { mkdtempSync, readFileSync } = require('fs');
+  const { tmpdir } = require('os');
+  const { join } = require('path');
+  const { sync: execa } = require('execa');
+
+  const dir = mkdtempSync(join(tmpdir(), 'msw-'));
+
+  execa(process.execPath, [
+    require.resolve('msw/cli'),
+    'init',
+    dir,
+    '--no-save',
+  ]);
+
+  const content = readFileSync(join(dir, 'mockServiceWorker.js'));
+
+  return content;
+};
+
+module.exports = class MswMixin extends Mixin {
+  configureBuild(webpackConfig) {
+    if (process.env.ENABLE_MOCKS !== 'true') {
+      webpackConfig.resolve = webpackConfig.resolve || {};
+      webpackConfig.resolve.alias = webpackConfig.resolve.alias || {};
+      webpackConfig.resolve.alias['hops-msw'] = false;
+      return;
```

Comment:
Why do we need this? With this change we create different bundles based on the env var `ENABLE_MOCKS`, which means we cannot use the production artifact to run the tests.
Since we're using a dynamic `import()` in the bootstrap method of the browser mixin, we shouldn't have bigger bundles anyway.